config/default/manager_webhook_patch.yaml

```
@@ -0,0 +1,23 @@
+        - containerPort: 9443
+          name: webhook-server
+          protocol: TCP
+        volumeMounts:
+        - mountPath: /tmp/k8s-webhook-server/serving-certs
+          name: cert
+          readOnly: true
+      volumes:
+      - name: cert
+        secret:
+          defaultMode: 420
+          secretName: webhook-server-certs 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)